docs/content/en/docs/migrate/strategy/_index.md

```diff
@@ -12,5 +12,259 @@ but the format and content is still evolving.
+such as Jenkins, Argo Workflows, Flux, and Tekton.
+
+### task
+
+Keptn v1 defines some specific types of tasks,
+each of which is translated to KLT resources
+that are appropriate for the activity:
+
+* A **deployment task** becomes a
+  [Deployment](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+  workload.
+  You can code
+  [KeptnTaskDefinition](../../yaml-crd-ref/taskdefinition.md)
+  and
+  [KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+  resources that are configured
+  to run either pre- or post-deployment tasks
+* An **evaluation task** becomes a
+  [KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+  resource.
+* All other standard tasks
+  (**action**, **approval**, **get-action**, **rollback**,
+  **release**, **test**)
+  as well as custom task types
+  that might be defined should be translated into
+  `KeptnTaskDefinition` resources.
+* The `key:value` **properties** for each Keptn v1 sequence
+  should be coded into the `KeptnTaskDefinition` resource
+  as appropriate.
+
+### SLIs
+
+Keptn v1
+[SLIs](https://keptn.sh/docs/1.0.x/reference/files/sli/)
+(Service Level Indicators)
+represent queries from the data provider
+such as Prometheus, Dynatrace, or Datadog,
+which is configured as a Keptn integration.
+
+When migrating to KLT, you need to define a
+[KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md)
+resource for the data provider you are using.
+
+The queries defined for the Keptn v1 SLIs
+should be translated into
+[KeptnMetric](../../yaml-crd-ref/metric.md)
+resources.
+Note that KLT allows you to support multiple data providers
+and multiple instances of each data provider for your SLI's
+whereas Keptn v1 only allows you to use one SLI per project.
+
+### SLOs
+
+KLT at this time does not support the full range
+of Quality Gates evaluations that are represented by
+[SLOs](https://keptn.sh/docs/1.0.x/reference/files/slo/).
+Facilities such as weighting of SLI's and scoring of the evaluation
+do not currently exist.
+However, simple evaluations of an SLI can be defined as
+[KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
+resources.
+
+### Remediation
+
+KLT does not currently support the same level of
+[remediations](https://keptn.sh/docs/1.0.x/reference/files/remediation/)
+as Keptn v1 does.
+KLT does provide limited "Day 2" facilities:
+
+* Any query that is possible for your data provider post-deployment
+  can be defined as a `KeptnMetricDefinition`;
+  this is then reported as a Keptn Metric.
+  Evaluation can be defined as a `KeptnEvaluationDefinition`.
+* `KeptnMetricsDefinition` resources can be retrieved and used
+  to implement the Kubernetes HorizontalPodAutoscaler (HPA),
+  which can detect the need for additional resources
+  (more pods, memory, disk space, etc)
+  and automatically add those resources to your configuration
+  based on the `ReplicaSet` resources you have defined
```

Comment:
maybe we could add that synce keptn clearly exposes metrics and crds related to each app, one could bring/design their own remediation ? Not sure really about what is there

Reply:
Maybe @thisthat can suggest appropriate remarks.  My understanding is that very few people are really using the remediation functionality of Keptnv1.  Mostly they want notification that something is wrong so they can go fix it.